Docs/text link (#484)
pluralsight/tva: website/docs/development/headless-styles/TextLink.mdx

## Usage

### Basic Text Link

<BasicTextLink />
<LiveExampleTabs
  fullCode={<BasicTextLinkFullPreview />}
  sandboxList={{
    chakra: 'https://codesandbox.io/s/basic-text-link-react-chakra-71898i',
    joy: 'https://codesandbox.io/s/basic-text-link-mui-dv200j',
    styled: 'https://codesandbox.io/s/basic-text-link-css-in-js-sc-j34zvf',
    svelte: 'https://codesandbox.io/s/basic-text-link-svelte-r816y1',
  }}
>
  <BasicTextLinkPreview />
</LiveExampleTabs>

> :::note

Reviewer: I don't think we need this. I would kill it.

`iconOptions` are provided only for configuring an icon to communicate an _external_ link's behavior.

:::

## Accessible Link Text

When using assistive technology, some users will jump directly to links.

> Therefore it is important that the link text makes sense and communicate its purpose without context (avoid phrases like “Click Here,” “More,” and “Read More”).

Reviewer: ```suggestion
Therefore, it is important the link text clearly communicates its purpose without context (i.e. avoid phrases like “Click Here,” “More", “Read More”, etc.).
```

> Link text should also be unique and easy to speak out loud.

Reviewer: ```suggestion
```
